File: AuctionServices/src/main/java/com/artauction/auctionservice/Dtos/PostSold/BuyerPostSoldDto.java
package com.artauction.auctionservice.Dtos.PostSold;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
public class BuyerPostSoldDto {
    private String title;
    private BigDecimal finalPrice;
    private boolean isPaid;
}

File: Project/Backend/src/ArtAuction.Application/DTOs/PostSold/BuyerPostSoldDto.cs
namespace ArtAuction.Application.DTOs.PostSold;

public class BuyerPostSoldDto
{
    public int ArtworkPostId { get; set; }
    public string Title { get; set; }
    public decimal FinalPrice { get; set; }
    public bool IsPaid { get; set; }
}
